src/main/scala/ai/privado/exporter/DataflowExporter.scala

```diff
@@ -70,23 +70,18 @@ class DataflowExporter(cpg: Cpg, dataflowsMap: Map[String, Path], taggerCache: T
     sinkSubCategory: String,
     ruleCache: RuleCache
   ): List[DataFlowSubCategorySinkModel] = {
-    def convertSink(sourceId: String, sinkId: String, sinkPathIds: List[String]) = {
-      val sinkIdAfterSplit = sinkId.split("#_#")
+    def convertSink(sourceId: String, sinkId: String, sinkPathIds: List[String], urls: mutable.Set[String]) = {
 
       // Special case for API type of nodes
-      val apiUrl = ruleCache.getRuleInfo(sinkIdAfterSplit(0)) match {
-        case Some(rule) if rule.nodeType.equals(NodeType.API) & sinkIdAfterSplit.size >= 2 =>
-          sinkIdAfterSplit(1).split(",").toList
-        case _ => List[String]()
-      }
+      val apiUrl = if (urls.size > 0) urls.toList else List[String]()
```

Comment:
`urls.empty` or `isEmpty` can be used

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/commit/4110826932d0e7852b80590ee5c5b98e91c22fdb